Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.h

```diff
@@ -39,6 +39,17 @@ namespace AzToolsFramework
         Q_OBJECT
 
     public:
+        enum CursorInputMode {
```

Comment:
update: Consider moving this definition outside of the class body too, it'll make it easier to use elsewhere if we need to in future

Reply:
where would I move this to? would it just be outside the class in the same file?

Reply:
Yeah I think that's good for now, it'll then be easier to use in other cases in future, doesn't have to be too coupled to the Qt input mapper